brooklyn-server/server-cli/src/main/java/org/apache/brooklyn/cli/ItemLister.java

```diff
@@ -98,88 +115,150 @@ public Void call() throws Exception {
             List<URL> urls = getUrls();
             LOG.info("Retrieving objects from "+urls);
 
-            // TODO Remove duplication from separate ListPolicyCommand etc
-            List<Class<? extends Entity>> entityTypes = getTypes(urls, Entity.class);
-            List<Class<? extends Policy>> policyTypes = getTypes(urls, Policy.class);
-            List<Class<? extends Enricher>> enricherTypes = getTypes(urls, Enricher.class);
-            List<Class<? extends Location>> locationTypes = getTypes(urls, Location.class, Boolean.FALSE);
+            Map<String, Object> result = MutableMap.of();
+            List<String> jsonList = new ArrayList<>();
 
-            Map<String, Object> result = ImmutableMap.<String, Object>builder()
-                    .put("entities", ItemDescriptors.toItemDescriptors(entityTypes, headingsOnly, "name"))
-                    .put("policies", ItemDescriptors.toItemDescriptors(policyTypes, headingsOnly, "name"))
-                    .put("enrichers", ItemDescriptors.toItemDescriptors(enricherTypes, headingsOnly, "name"))
-                    .put("locations", ItemDescriptors.toItemDescriptors(locationTypes, headingsOnly, "type"))
-                    .put("locationResolvers", ItemDescriptors.toItemDescriptors(ImmutableList.copyOf(ServiceLoader.load(LocationResolver.class)), true))
-                    .build();
+            if (!jars.isEmpty()) {
+                // TODO Remove duplication from separate ListPolicyCommand etc
+                List<Class<? extends Entity>> entityTypes = getTypes(urls, Entity.class);
+                List<Class<? extends Policy>> policyTypes = getTypes(urls, Policy.class);
+                List<Class<? extends Enricher>> enricherTypes = getTypes(urls, Enricher.class);
+                List<Class<? extends Location>> locationTypes = getTypes(urls, Location.class, Boolean.FALSE);
 
-            String json = toJson(result);
+                result = ImmutableMap.<String, Object>builder()
+                        .put("entities", ItemDescriptors.toItemDescriptors(entityTypes, headingsOnly, "name"))
+                        .put("policies", ItemDescriptors.toItemDescriptors(policyTypes, headingsOnly, "name"))
+                        .put("enrichers", ItemDescriptors.toItemDescriptors(enricherTypes, headingsOnly, "name"))
+                        .put("locations", ItemDescriptors.toItemDescriptors(locationTypes, headingsOnly, "type"))
+                        .put("locationResolvers", ItemDescriptors.toItemDescriptors(ImmutableList.copyOf(ServiceLoader.load(LocationResolver.class)), true))
+                        .build();
+                jsonList.add(toJson(result));
+            } else if (!yaml.isEmpty()) {
+                LocalManagementContext lmgmt = new LocalManagementContext();
```

Comment:
This will read config from the user's `brooklyn.properties`. Better initialize it with an empty state using `new LocalManagementContext(BrooklynProperties.Factory.newEmpty());`